src/covid_shared/ihme_deps.py

```diff
@@ -0,0 +1,46 @@
+"""Wrappers for IHME specific dependencies.
+
+This module explicitly declares and wraps IHME specific code bases
+to prevent CI failures at import time.
+
+"""
+import importlib
+
+
+def _lazy_import_callable(module_path: str, object_name: str):
+    """Delays import errors for callables until called."""
+    try:
+        module = importlib.import_module(module_path)
+        return getattr(module, object_name)
+    except ModuleNotFoundError:
+        def f(*args, **kwargs):
+            raise ModuleNotFoundError(
+                f"No module named '{module_path}' and so we cannot find '{object_name}'. "
+                f"Ensure you have a file in your home directory at '~/.pip/pip.conf' with contents\n\n"
+                f"[global]\n"
+                f"extra-index-url = https://artifactory.ihme.washington.edu/artifactory/api/pypi/pypi-shared/simple\n"
+                f"trusted-host = artifactory.ihme.washington.edu/artifactory/api/pypi/pypi-shared\n\n"
+                f"and run 'pip install {module_path.split('.')[0]}'")
+        return f
+
+
+try:
```

Comment:
Should lines 27+ all be indented as part of the `_lazy_import_callable` function?

Reply:
No.  Lines 27+ are replacing functions/classes we would import from internal libraries with callables that won't error until called.